models/modelUsers.js

```diff
@@ -0,0 +1,11 @@
+const connection = require("../db/connection")
+
+exports.fetchUsers = () => {
+    
+    return connection
+    .query ('SELECT * FROM users;').then(({rows}) => {
+        console.log(rows)

```

Comment:
try not to leave console logs floating about in what could be considered production code, this should be removed